README.rst

```diff
@@ -29,6 +31,13 @@ Features
 
 .. _Issue 67: https://github.com/aiven/karapace/issues/67
 
+Compatibility details
+---------------------
+
+Karapace is compatible with Schema Registry 6.1.1 on API level. When a new version of SR is released, the goal is
+to support it in a reasonable time. Karapace supports all operations in the API.
+The goal is that even the error messages are the same as in Schema Registry, which cannot be always fully guaranteed.
```

Comment:
Should we mention also about compatibility status with Confluent's REST Proxy in addition to Schema Registry?